Jenkinsfile

```diff
@@ -54,7 +56,21 @@ def sh = { String cmd ->
 	}
 }
 
-node('windows') {
+def bootstrap = { String node ->
+	bootstrapCmd = node == windows ? bat(pscmd('.\\make bootstrap')) : make('bootstrap')
+
+	try {
+		bootstrapCmd
+	} catch(error) {
+		echo "bootstrap failed; wiping 'vendor' directory and trying again..."
+		retry(1) {
```

Comment:
Good inquiry to check @Joshua-Anderson -- it appears `retry` does the right thing and only attempts the given number of times; see https://ci.deis.io/job/Deis/job/workflow-cli/job/PR-176/6/console